photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -206,19 +206,19 @@ private void createStreams() {
     private void recreateStreamResultConsumers() {
         streamResultConsumers.add(
                 (frame, tgts) -> {
-                    if (frame != null) inputFrameSaver.accept(frame.colorImage);
+                    inputFrameSaver.accept(frame.colorImage);
                 });
         streamResultConsumers.add(
                 (frame, tgts) -> {
-                    if (frame != null) outputFrameSaver.accept(frame.processedImage);
+                    outputFrameSaver.accept(frame.processedImage);
                 });
         streamResultConsumers.add(
                 (frame, tgts) -> {
-                    if (frame != null) inputVideoStreamer.accept(frame.colorImage);
+                    inputVideoStreamer.accept(frame.colorImage);
                 });
         streamResultConsumers.add(
                 (frame, tgts) -> {
-                    if (frame != null) outputVideoStreamer.accept(frame.processedImage);
```

Comment:
is it always guaranteed the frame will be non-null now? I generally like the intent (sources must always supply a frame, even if that means some hardcoded graphic explaining why there's no real frame). Just wanna make sure we've covered the bases.